src/main/java/io/github/queerbric/queerbrics/QueerbricsMain.java

```diff
@@ -1,10 +1,19 @@
 package io.github.queerbric.queerbrics;
 
+import io.github.queerbric.queerbrics.registry.QueerbricsItems;
 import net.fabricmc.api.ModInitializer;
+import net.minecraft.util.Identifier;
 
 public class QueerbricsMain implements ModInitializer {
+	public static String modid = "queerbrics";
+	
+	// Returns an Identifier of "modid:key" (ex: "queerbrics:rainbow_brick")
+	public static Identifier getId(String key) {
+		return new Identifier(modid, key);
+	}
+	
 	@Override
 	public void onInitialize() {
-
+		QueerbricsItems.registerAll();
```

Comment:
would prefer just `register()`

Reply:
IMO not specific enough.
"register" can mean "register a registerable object" or "register an item"
but
"registerAll" clearly means "register all the objects in the class.

Reply:
Changed to init() per conversation on Discord.